public/components/ctas/8.html

```diff
@@ -0,0 +1,26 @@
+<aside class="relative overflow-hidden text-gray-300 bg-gray-900 lg:flex justify-center">
+    <div class="w-full p-12 text-center lg:w-2/3 sm:p-16 lg:p-24">
```

Comment:
Not a fan of `w-full lg:w-2/3`. The child element uses `max-w-xl` so why are these needed?